### ISSUE TITLE
Struct value cannot have same name as struct type

### ISSUE THREAD

**Issue by pema99** [ispc/ispc#2278]
This program fails to compile with ISPC:
```c
struct TestStruct {
	int a;
};

int main() {
	struct TestStruct TestStruct;
	return 0;
}
```
```
test.ispc:6:2: Error: Multiple types provided for declaration.
 struct TestStruct TestStruct;
 ^^^^^^^^^^^^^^^^^
```
But compiles fine using `gcc`. Is the difference to C behavior intentional?

**Comment by pbrubaker** [comment]
Possibly related to https://github.com/ispc/ispc/issues/2351

**Comment by nurmukhametov** [comment]
Fixed in v1.25.0 probably by https://github.com/ispc/ispc/pull/2972
